Source/Fuse.Launcher.InterApp/InterApp/InterAppLauncher.uno

```
@@ -36,5 +36,22 @@ namespace Fuse.LauncherImpl
 				iOSDeviceInterop.LaunchUriiOS(uri.AbsoluteUri);
 			}
 		}
+
+		public static void LaunchApp(string uri)
+		{
+			if defined(Android)
+			{
+				Permissions.Request(Permissions.Android.INTERNET);
+				try {
+					AndroidDeviceInterop.LaunchApp(_actionView, uri);
+				} catch (Exception e) {
+					// ignore this error. Keeps behaviour in line with iOS
```

Maybe we should report the exception to logcat at least?